x/programs/rust/wasmlanche_sdk/src/store/mod.rs

```diff
@@ -139,20 +157,26 @@ fn get_field_as_bytes(ctx: &Context, name: &[u8]) -> Result<Vec<u8>, StorageErro
     let bytes_ptr = bytes_ptr as *mut u8;
 
     // Take ownership of those bytes grabbed from the host. We want Rust to manage the memory.
-    let bytes = unsafe { Vec::from_raw_parts(bytes_ptr, bytes_len as usize, bytes_len as usize) };
+    let bytes = unsafe {
+        Vec::from_raw_parts(
+            bytes_ptr,
+            bytes_len.try_into().unwrap(),
+            bytes_len.try_into().unwrap(),
+        )
+    };
     Ok(bytes)
 }
 
-/// Gets the field [name] from the host and returns it as a ProgramValue.
-fn get_field<T>(ctx: &Context, name: &str) -> Result<T, StorageError>
+/// Gets the field [name] from the host and returns it as a `ProgramValue`.
+fn get_field<T>(ctx: Context, name: &str) -> Result<T, StorageError>
 where
     T: DeserializeOwned,
 {
     let bytes = get_field_as_bytes(ctx, name.as_bytes())?;
     from_slice(&bytes).map_err(|_| StorageError::InvalidBytes)
 }
 
-/// Gets the correct key to in the host storage for a [map_name] and [key] within that map  
+/// Gets the correct key to in the host storage for a [`map_name`] and [key] within that map  
```

Comment:
why ` `` ` and `[]`?

Reply:
`clippy --fix`... but good call out, these docs are actually not formatted properly.